src/lava/proc/io/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
We currently do not use init files anywhere for some reason, correct? Do we really need this?